CONTRIBUTING.md

```diff
@@ -39,9 +39,9 @@ branch points. To check how much coverage the tests have, you will need.
 
 When running:
 
-    nosetests --with-coverage --cover-package=dipy
+    coverage run -m pytest -s --doctest-modules --verbose dipy
```

Comment:
Also get `Coverage.py warning: --include is ignored because --source is set (include-ignored)` when I run this locally.